src/views/admin/PendingAccounts.vue

```diff
@@ -186,4 +231,7 @@
    text-overflow: ellipsis
   table
    table-layout: fixed
+  #discard
+    background-color: #cc0f35 !important
+    border-color: #cc0f35 !important
```

Comment:
why the `!important` ?

Reply:
because the old css in contains a lot of unnecessary !important that we need to remove. i already fixed the button color by removing few of them